boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7.yaml

```diff
@@ -14,3 +14,4 @@ supported:
 testing:
   ignore_tags:
     - isotp
+    - pwm
```

Comment:
I just noticed you are ignoring all `pwm` tests/samples for this board - how come?

Reply:
As mentioned into the PR message, pwm_api tests use prohibited values that are not hardware supported by our PWM controller. That's why with @aaillet we decided to put it as "ignored" in order to not block twister test suite.
Should we propose a fix into the pwm_api test to add a particular case for this driver ? What is your advise ?
Thanks

Reply:
> As mentioned into the PR message, pwm_api tests use prohibited values that are not hardware supported by our PWM controller. That's why with @aaillet we decided to put it as "ignored" in order to not block twister test suite.
> Should we propose a fix into the pwm_api test to add a particular case for this driver ? What is your advise ?

The common PWM tests should be driver agnostic. I'd submit an issue - or even a PR, if you have a proposal for a fix.